compiler/ballerina-lang/src/main/java/io/ballerina/projects/ResourceConfig.java

```diff
@@ -64,11 +63,8 @@ public String name() {
         return name;
     }
 
-    public Optional<byte[]> content() {
-        if (content == null) {
-            return Optional.empty();
-        }
-        return Optional.of(content);
+    byte[] content() {
```

Comment:
I'm not sure if this is related to the IDL change? Can you check with the author and verify if it is OK to change?

Reply:
This change done in client declaration PR
https://github.com/ballerina-platform/ballerina-lang/pull/37584/files#diff-639d8db2b38716743cfcf48a52e2de8c9316b1339344de9549f10411583c692b